contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,522 @@
+  Kubernetes better support of high performance applications that can utilize raw block devices directly for their storage. 
+  Block volumes are critical to applications like databases (MongoDB, Cassandra) that require consistent I/O performance
+  and low latency. For mission critical applications, like SAP, block storage is a requirement. 
+  
+  For applications that use block storage natively (like MongoDB) no additional configuration is required as the mount path passed
+  to the application provides the device which MongoDB then uses for the storage path in the configuration file (dbpath).
+  
+  Specific use cases around improved usage of storage consumption are included in the use cases listed below as follows:
+  * An admin wishes to expose a block volume to be consumed as a block volume for the user
+  * A user wishes to be specific about the storage consumption and intentionally request a block device
+  * A user wishes to utilitze block storage to fully realize the performance of an application tuned to using block devices
+  * A user wishes to utilize raw block devices for consumption from a virtual machine
+  * A user wishes to specify an inline volume as a block device in their pod
+  Future use cases include dynamically provisioning and intelligent discovery of existing devices, which this proposal sets the 
+  foundation for more fully developing these methods. 
+  
+  It is importnant to note that when a PV is bound, it is either bound as a raw block device or formatted with a filesystem. Therefore, 
+  the PVC drives the request and intended usage of the device by specifying the volumeMode as part of the API. This design lends itself
+  to future support of dynamic provisioning by also letting the request intiate from the PVC defining the role for the PV. It also allows
+  flexibility in the implementation and storage plugins to determine their support of this feature.
+  
+  
+    
+# Design Overview
+
+  The proposed design is based on the idea of leveraging well defined concepts for storage in Kubernetes. The consumption and 
+  definitions for the block devices will be driven through the PVC and PV and Storage Class definitions. Along with Storage
+  Resource definitions, this will provide the admin with a consistent way of managing all storage. 
+  The API changes proposed in the following section are minimal with the idea of defining a volumeMode to indicate both the definition
+  and consumption of the devices. Since it's possible to create a volume as a block device and then later consume it by provisioning
+  a filesystem on top, the design requires explicit intent for how the volume will be used.
+  The additional benefit of explicitly defining how the volume is to be consumed will provide a means for indicating the method
+  by which the device should be scrubbed when the claim is deleted, as this method will differ from a raw block device compared to a 
```

Comment:
I thought we wanted to remove this whole scrubbing discussion and replace it with one sentence about the reclaim policy.

Reply:
done